cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -925,13 +930,119 @@ def make_resource(self, data, **kwargs):
         return Timeouts(**data)
 
 
+class BudgetNotificationSchema(BaseSchema):
+    """Represent the schema of each notification under the NotificationsWithSubscribers field of a budget."""
+
+    notification_type = fields.Str(
+        metadata={"update_policy": UpdatePolicy.SUPPORTED},
+        validate=validate.OneOf(["ACTUAL", "FORECASTED"]),
+    )
+    comparison_operator = fields.Str(
+        metadata={"update_policy": UpdatePolicy.SUPPORTED},
+        validate=validate.OneOf(["GREATER_THAN", "EQUAL_TO", "LESS_THAN"]),
+    )
+    threshold = fields.Int(required=True, metadata={"update_policy": UpdatePolicy.SUPPORTED})
```

Comment:
Does `threshold` require to be an interger? Can it be float?

Reply:
It should be a float. Updated!